src/components/FeatureViews/Scheduler/DisplaySessionView.js

```diff
@@ -106,6 +109,22 @@ function DisplaySessionView({ course, session, handleToggleEditing }) {
         setEditSelection(event.target.value);
     };
 
+    const handleUnenroll = event => {
+        event.preventDefault();
+       setUnenroll(true);
+    };
+
+    const closeUnenrollDialog = (toUnenroll) => event => {
+        event.preventDefault();
+        setUnenroll(false);
+        if(toUnenroll){
+            const courseID = course.course_id,
+                studentID = course.roster[0],
+                enrollmentID = enrollments[course.roster[0]][course.course_id].enrollment_id;
```

Comment:
Add a comment to clarify that this only happens if there is one student in the roster.